src/pydvl/value/sampler.py

```diff
@@ -257,7 +265,7 @@ def weight(cls, n: int, subset_len: int) -> float:
         return float(2 ** (n - 1)) if n > 0 else 1.0
 
 
-class UniformSampler(StochasticSampler[T]):
+class UniformSampler(PowersetSampler[T], StochasticSamplerMixin):
```

Comment:
@kosmitive Are you sure you don't want to swap the order of inheritance here? Recall that python resolves methods in the order you use here. If you want to be able to initialise UniformSampler with a seed, which would be an argument for the mixin, you need to ensure that the `__init__` of the mixin is called first. See the example [in my other comment](https://github.com/appliedAI-Initiative/pyDVL/pull/396#discussion_r1310585383)

Reply:
Found a different solution, where it doesn't depend on the order.